### DISCUSSION TITLE
IP Banned by GitHub

### DISCUSSION THREAD

**Discussion by RomkaKharkov** [community/community#42913] [Other Feature Feedback, Questions, & Ideas]
Hi,

Please move this thread to right category (if i select wrong one).

Our ip's banned by github (we even can't access website) , seems because we order subnets from Iran company (in October 2021)

How we can solve this issue ?

**Comment by jfullstackdev** [comment]
there are certain places that GitHub is not allowed but generally, you can have a VPN for that, like in China, the VPN is working

**Reply by RomkaKharkov** [reply]
@jdevstatic 
Yep, i see , many services not like IRAN, but.... this subnet reassigned with RIPE to our company which not IRAN yet, and this subnets not point to IRAN anymore... 

We buy this subnets with *official way* and just want someone from github update his block-lists based on *official* RIPE data.

----------
Oct 21, 2021, 13:53 GMT+2

Dear colleagues,

The RIPE NCC has completed the transfer of the following resources from
Favapayam Pars Co., Ltd (ir.favapayam) to HZ Hosting Ltd (bg.hzhosting):

IPv4 allocation: 185.174.133.0/24
IPv4 allocation: 185.174.135.0/24
----------

**Reply by RomkaKharkov** [reply]
> there are certain places that GitHub is not allowed but generally, you can have a VPN for that, like in China, the VPN is working

Its possible to unblock this ips ?

**Comment by hellodarkness** [comment]
As a short-term solution could you use a foreign VPN, or perhaps even Tor browser?

**Reply by RomkaKharkov** [reply]
> As a short-term solution could you use a foreign VPN, or perhaps even Tor browser?

Its not a solution yet, we are Hosting Provider which provide VPS (and other services) to customers, with this ip ranges, should we install VPN to each new server? :) sure no .... we just ask to update block lists from official place, this subnets not IRAN's anymore!

**Reply by hellodarkness** [reply]
Ah, sorry I didn't realize you were a hosting provider. I assumed you were talking about your office IPs. Yes, I can understand your frustration there!
